src/System.ComponentModel.Annotations/tests/CustomValidationAttributeTests.cs

```diff
@@ -69,6 +69,26 @@ public static void Constructor(Type validatorType, string method)
             Assert.Equal(method, attribute.Method);
         }
 
+        [Fact]
+        public void FormatErrorMessage_NoError_ContainsName()
```

Comment:
Nit: Just checking - did you mean this to say "NoError"? The test is about what happens to the error message? The test below checks that validation fails - so did you mean "validation passes" here? If so maybe add that assert or rename the tests.